lib/adapters/AbstractAdapter.js

```diff
@@ -209,6 +209,11 @@ class AbstractAdapter extends AbstractReaderWriter {
 	}
 
 	_write(resource) {
+		if (!resource.getPath().startsWith(this._virBasePath)) {
+			throw new Error(`The path of the resource '${resource.getPath()}' does not starts with ` +
```

Comment:
does not **start**

Reply:
Good catch! Fix is prepared in https://github.com/SAP/ui5-fs/pull/457